changes/README.md

```diff
@@ -0,0 +1,50 @@
+# Writing Changes
+
+All notable changes to this project should be documented. Changes should be written to files in
+this directory, which will be merged into the changelog during release via `compact-changelog.sh`.
+
+## Change Types
+
+The following change types are supported. Changes will be listed by type in the changelog, and the
+types of changes contained in a release determine the release type. If there are any changes
+requiring a minor release, a minor release should be performed. If there are only changes that
+require a patch release, then a patch release should be performed instead.
+
+| Change Type | Filename Prefix | Description | Release Type |
+|-------------|-----------------|-------------|--------------|
+| Added       | `add_`          | New features | minor |
+| Changed     | `change_`       | Changes in existing functionality | minor |
+| Removed     | `remove_`       | Features that have been removed | minor |
+| Fixed       | `fix_`          | Bug fixes and performance improvements | patch |
+
+## File Names
+
+The file name must begin with a prefix depending on the change type (see above), followed by an
+underscore. e.g. `fix_saving_samples.txt` or `add_GLT-1234.txt`. The portion after the underscore
+is not important, but something descriptive makes them easier to find if they need to be modified.
+
+## Message Format
+
+Each change must be in a separate file, but a single change may contain multiple lines of text. A
+bullet point will be added before the first line when writing to the changelog, so the first line
+in a change file should not have a bullet point. Subsequent lines may use bullet points, which will
+be indented as a second-level list. For example, the following is a well-formatted change, which
+could be named `add_field_x.txt`:
```

Comment:
Worth mentioning that change messages can be written in Markdown (though there's no guarantee that the script will correctly parse more complex features)?

Reply:
👍 Updated to say that it's markdown